src/styles/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      margin-right: 0;
+      background-image: url(../images/star.svg);
```

Comment:
According to the checklist, we should avoid specifying the background image in a modifier like `&:last-child`. Instead, set the default background image for the `.stars__star` element, and then override it with the active star image for the appropriate modifiers.